### DISCUSSION TITLE
Interested in contribution

### DISCUSSION THREAD

**Discussion by TheAwiteb** [SeaQL/summer-of-code#17] [General]
Hello, I am Awiteb, a Rust programmer and a university student who is interested in contributing to open source projects. Indeed, I am interested in working on the [fourth idea](https://github.com/SeaQL/summer-of-code/blob/main/2023/README.md#4-a-media-centric-cms-based-on-seaorm--seaography). I have some sea-orm contributions which are:
- https://github.com/SeaQL/sea-orm/issues/1222
- https://github.com/SeaQL/sea-orm/issues/1490
- https://github.com/SeaQL/sea-orm/pull/1492
- https://github.com/SeaQL/sea-orm/pull/1484

My discord is `Awiteb#6210`

**Comment by billy1624** [comment]
Hi @TheAwiteb, thanks for your interest in contributing to SeaQL!! We were a mentor organization in GSoC 2022. However, we were not accepted into GSoC 2023.

With that said, we will offer our own internship program, it will be the 3rd cohort in 2023. An announcement will be made shortly, stay tuned.

**Comment by billy1624** [comment]
Hey @TheAwiteb, please check here for the internship details :)
https://www.linkedin.com/jobs/view/3519852307/
